ansible_rulebook/builtin.py

```diff
@@ -76,6 +76,9 @@ async def debug(event_log, **kwargs):
     print("kwargs:")
     pprint(kwargs)
     print(get_horizontal_rule("="))
+    print("facts:")
+    pprint(lang.get_facts(kwargs["source_ruleset_name"]))
```

Comment:
I tested this code with the rulebook from the Jira issue, the facts are an empty array []. Is it expected?

Reply:
@bzwei The only time the rule engine stores anything in permanent storage is when you do set_facts a good example is this one https://github.com/ansible/ansible-rulebook/blob/4a3c9930c153b138df56311147b3a373fac84378/tests/examples/04_set_fact.yml#L14 but you would have to change the following action from print_event to debug to see the data.